src/components/RichTextEditor/serializer/utils/serializeLeafToHtml.ts

```diff
@@ -12,7 +12,7 @@ import escapeHtml from 'escape-html';
 
 export const serializeLeafToHtml = (node: TText): string => {
     let string = escapeHtml(node.text);
-    string = string.replace('\n', '<br />');
+    string = string.replace(/\n/g, '<br />');
```

Comment:
```suggestion
    string = string.replaceAll(/\n/g, '<br />');
```
So ESLint doesn't complains after merging https://github.com/Frontify/fondue/pull/1394